pyshield/radiation/radiation_state.py

```diff
@@ -0,0 +1,404 @@
+from dataclasses import InitVar, dataclass, field, fields
+from typing import Any, Dict, Mapping
+
+import xarray as xr
+
+import ndsl.dsl.gt4py_utils as gt_utils
+from ndsl import GridSizer, Quantity, QuantityFactory
+from ndsl.constants import X_DIM, Y_DIM, Z_DIM, Z_INTERFACE_DIM
+from ndsl.dsl.typing import Float
+from ndsl.types import NumpyModule
+
+
+@dataclass()
+class RadiationState:
```

Comment:
We need `States` to become a `NDSL` natively orchestrateable thing in the near future